index.html

```diff
@@ -281,7 +282,39 @@ <h2>Skills</h2>
         </div>
 
     </section>
+    <div class="div"><br><br><br></div>
+    <!-- Contact Us Section -->
+    <div class="div">
+        <section class="contact">
+            <div class="contact-items">
+                <h2>Contact me</h2>
+            </div>
+            <div class="contact-items">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
 
+            </div>
+            <div class="contact-items">
+                <form action="https://formspree.io/f/{form_id}" method="post" class="form-flex">
+                    <div class="form-flex-item ">
+                        <input type="text" placeholder="Lorem Ipsum" maxlength="30" name="userName">
+                    </div>
+                    <div class="form-flex-item ">
+                        <input type="emial" placeholder="loremipsum@gmail.com" name="emial">
+                    </div>
+                    <div class="form-flex-item textarea">
+                        <textarea name="" id="" cols="30" rows="12" maxlength="500" required name="text">Write your message here...
```

Comment:
- [ ] when i filled the form it displayed form not foumd here the screenshot

![form not sun](https://user-images.githubusercontent.com/92458236/235795023-63c69d74-4bba-40af-8a40-267b7f0c594a.jpg)

- [ ] kindly consider to add reqired attribut to validate the form , be confirm that the form is submitted successfully with valid input values. All input fields have the required attribute to activate the validation on submit  After submitting the form it should display  comfirmation page that the form was submitted successfully.kindly fix this